includes/wikia/parser/templatetypes/TemplateTypesParser.class.php

```diff
@@ -37,6 +38,36 @@ public static function onFetchTemplateAndTitle( &$text, &$finalTitle ) {
 	}
 
 	/**
+	 *
+	 * @param string $text - template content
+	 * @param Title $finalTitle - template title object
+	 *
+	 * @return bool
+	 */
+	public static function onFetchTemplateAndTitleForTables( &$text, &$finalTitle ) {
+		wfProfileIn( __METHOD__ );
```

Comment:
I think this could preferrably be done in hook setup: 
https://github.com/Wikia/app/pull/9224/files#diff-5252738fa9bbecd8be3a432e08189a04R26
https://github.com/Wikia/app/pull/9224/files#diff-5252738fa9bbecd8be3a432e08189a04R29